lib/humidifier/aws_adapters/base.rb

```diff
@@ -31,6 +31,12 @@ def update(payload)
         try_valid { client.update_stack(payload.update_params) }
       end
 
+      # Upload a CFN stack to S3 so that it can be referenced via template_url
+      def upload(payload)
+        Humidifier.config.ensure_upload_configured!(payload)
+        upload_object(payload, "#{Humidifier.config.s3_prefix}#{payload.identifier}.json")
```

Comment:
I think we should at least update the example [here](https://github.com/localytics/humidifier/pull/48/files#diff-9f071b217da54d7034194a75c6b28257R13) to include a trailing slash then